src/encord_active/lib/embeddings/utils.py

```diff
@@ -102,6 +102,10 @@ class Embedding2DSchema(IdentifierSchema, PointSchema2D):
     label: Series[str]
 
 
+class Embedding2DScoreSchema(Embedding2DSchema):
+    score: Series[float]
+
+
```

Comment:
Depending on whether this PR or #433 gets merged first, we'll have to move this definition to `src/encord_active/lib/embeddings/types.py`.

Reply:
You mean all the schemas in this file right